pipelines/internal/commands/run/run.go

```diff
@@ -180,30 +181,55 @@ func Invoke(ctx context.Context, service *genomics.Service, project string, argu
 	}
 	fmt.Printf("%s\n", encoded)
 
+	if *attempts < 1 {
+		return fmt.Errorf("The number of attempts must be > 1")
```

Comment:
I'd drop this, if they specify --atempts=0 it's not really worth complaining about

If you do want to keep it, you need to change the error string to be lowercase and correct, eg: "attempts must be non-zero"